### PR TITLE
[Review Needed]Pytest Style coordinates/test_crd.py

### PR DESCRIPTION
**Pull request by utkbansal**
Fixes #

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: testsuite/MDAnalysisTests/coordinates/test_crd.py [hunk 1]
```diff
@@ -89,23 +82,23 @@ class TestCRDWriterMissingAttrs(object):
                  'tempfactors': 0.0,
                  }
 
-    def _check_warns(self, missing_attr):
+    @pytest.mark.parametrize('missing_attr', req_attrs)
```
Comment (richardjgowers) [inline testsuite/MDAnalysisTests/coordinates/test_crd.py]: The tests are failing because parametrizing with a dict causes an unpredictable order for xdist

Comment (orbeckst) [inline testsuite/MDAnalysisTests/coordinates/test_crd.py]: Can you used a `collections.OrderedDict`?

Comment (richardjgowers) [inline testsuite/MDAnalysisTests/coordinates/test_crd.py]: Yeah sure, that'd work too.  Just something with a deterministic order, else "different" tests get collected by two processes.

**Comment by orbeckst** [comment]
(The tests passed, just the one encore test flaked.)

**Pull request merged**